styles/style.css

```
@@ -16,7 +16,14 @@
   font-display: swap;
 }
 /*general*/
+html {
+  height: 100%;
+}
```

Непомню, объясняли ли вам блочную модель на лекции, то лучше заранее поставить допустимые глобальные свойства из критерия [#CSS-2](https://up.htmlacademy.ru/htmlcss/32/criteries#CSS-02)
```suggestion
*,
*::before,
*::after {
  box-sizing: border-box;
}

img {
  max-width: 100%;
  object-fit: contain;
}

html {
  height: 100%;
}
```